BraveWallet/Preview Content/MockKeyringService.swift

```diff
@@ -11,14 +11,19 @@ import BraveCore
 ///
 /// - note: Do not use this directly, use ``CryptoKeyringStore.previewStore``
 class MockKeyringService: BraveWalletKeyringService {
-  private let keyring: BraveWallet.KeyringInfo = .init()
+  private var keyrings: [BraveWallet.KeyringInfo] = []
   private var privateKeys: [String: String] = [:]
   private var password = ""
   // Not a real phrase, has a duplicated word for testing
   private let mnemonic = "pass entire pelican lock repair desert entire cactus actress remain gossip rail"
   private var observers: NSHashTable<BraveWalletKeyringServiceObserver> = .weakObjects()
   private var selectedAccount: BraveWallet.AccountInfo?
   
+  init() {
+    let defaultKeyring: BraveWallet.KeyringInfo = .init(id: BraveWallet.DefaultKeyringId, isDefaultKeyringCreated: true, isLocked: false, isBackedUp: true, accountInfos: [])
+    keyrings.append(defaultKeyring)
```

Comment:
Why are we creating a keyring by default?

Reply:
Ah I see, originally we had an empty keyring by default. Then yeah I suppose we'd want to do something similar here